.github/workflows/test.yml

```diff
@@ -226,7 +229,7 @@ jobs:
       - name: "chore(health): check up on recent changes to the health score"
         uses: slackapi/slack-health-score@v0.1.1
         with:
-          codecov_token: ${{ secrets.CODECOV_TOKEN }}
+          codecov_token: ${{ secrets.CODECOV_API_TOKEN }}
```

Comment:
This was changed to be an API token and has been added as a secret- https://github.com/slackapi/slack-health-score/?tab=readme-ov-file#api-tokens

Reply:
Oh and this was changed because the standard upload token was being used: https://github.com/slackapi/slack-github-action/actions/runs/12173268802/job/33953505599#step:31:257

I don't think the testing workflow in this PR will show an update due to `pull_request_target`, but I'm optimistic with this change [working here](https://github.com/zimeg/slack-github-action/actions/runs/12321990320/job/34394566023#step:31:29) 🙏